src/components/interactive-builder/modals/question/question-form/common/sortable-tag/sortable-tag.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { useTranslation } from 'react-i18next';
+
+import { IconButton } from '@carbon/react';
+import { Draggable, WarningAltFilled } from '@carbon/react/icons';
+
+import styles from './sortable-tag.scss';
+
+interface SortableTagProps {
+  id: string;
+  text: string;
+  onDelete?: () => void;
+  isInvalid?: boolean;
+}
+
+export const SortableTag: React.FC<SortableTagProps> = ({ id, text, onDelete, isInvalid }) => {
+  const { t } = useTranslation();
+  const { attributes, listeners, setNodeRef, transform, transition, isDragging } = useSortable({
+    id,
+  });
+
+  const dynamicStyle = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+
+  return (
+    <div
+      ref={setNodeRef}
+      style={dynamicStyle}
+      className={`${styles.sortableTagWrapper} ${isDragging ? styles.dragContainerWhenDragging : ''}`}
```

Comment:
You can use the `classNames` library for instances like this:
```suggestion
className={classNames(styles.sortableTagWrapper, {
            [styles.dragContainerWhenDragging]: isDragging
          })}
```